src/pages/en/agents/what-is-an-agent.md

```
@@ -56,6 +56,4 @@ Execution delegates are part of the agent's operational state, not ephemeral app
 - **[Read Agent Data](/agents/read-agent-data)** — Verify registration and inspect agent identity on-chain
 - **[Run an Agent](/agents/run-an-agent)** — Set up an executive profile and delegate execution
 - **[Agent Finance](/agents/agent-finance)** — Capitalize the agent through its own onchain token
-- **[Agent Commerce](/agents/agent-commerce)** — Let the agent earn revenue and transact autonomously
-
-*Maintained by [Metaplex](https://github.com/metaplex-foundation) · Last verified March 2026*
+- **[Agent Commerce](/agents/agent-commerce)** — Let the agent earn revenue and transact autonomously
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)